org.mixedrealitytoolkit.spatialmanipulation/CHANGELOG.md

```diff
@@ -2,11 +2,32 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [4.0.0-development] - 2024-05-08
+## [4.0.0-development.pre.1] - 2024-06-18
+
+### Added
+
+* MRTK3 controllerless prefabs.
+* Added ObsoleteHandInteractionExamples scene that still uses deprecated MRTK3 rig.
+* Added XRI3+ exclusive unity-tests.
+* Added new TrackedPoseDriver lookup.
 
 ### Changed
 
-* Updated package com.unity.xr.interaction.toolkit to 3.0.1
+* Updated package com.unity.xr.interaction.toolkit to 3.0.3
+* Updated Unity Editor to 2022.3.7f1
+* Updated all scenes to consume new MRTK3 controllerless prefabs.
+* Updated existing unity-tests.
+* Updated code to support new XRI3 patterns.
+
+### Deprecated
+
+* MRTK3 controller-based prefabs marked as obsolete and deprecated.
+* Controller lookup deprecated.
+
+### Removed
+
+* Removed obsolete methods.
+* Removed legacy dialog*.
```

Comment:
Looks like the same changelog has been copied to all updated CHANGELOG.md files. Is that accurate? Was something removed from this package when legacy dialog was removed? Are there any scenes in this package (or any packages) that were updated to the new prefabs? Some entries are fairly generic and could be in all changelogs, but others seem more targeted.

Reply:
@ms-RistoRK , @keveleigh is correct.  The changelogs should only include information relative to the package.

Reply:
@keveleigh, @AMollis :  Where do you recommend me to document the "* Updated all scenes to consume new MRTK3 controllerless prefabs." change since the scenes are not located inside any package?

Reply:
@ms-RistoRK ... The samples aren't delivered via a package.  So, the PR / commit notes are good enough for sample changes

Reply:
Changelogs updated in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/781/commits/72870d1f1c991e3f323eb415967375f3b87c7538 , resolving.